src/commands/schema/push.js

```diff
@@ -52,13 +52,15 @@ class PushSchemaCommand extends SchemaCommand {
         if (json.error) {
           this.error(json.error.message);
         }
-        if (!json.diff) {
-          this.log("No changes to push");
-          this.exit(0);
+        let msg = "Accept and push changes?"
+        if (json.diff) {
+          this.log(`Proposed diff:\n`);
+          this.log(json.diff);
+        } else {
+          this.log("No logical changes to push");
```

Comment:
Idk if "logical" will resonate with users. Perhaps "No schema changes to push" or "No logical schema changes to push"?

Reply:
🤔 I think this is a word in common developer usage that people will understand, either immediately or with a moment's thought. Maybe I'll change it to "No logical changes. Push new file contents anyway?". Does that sound better?

Reply:
> No logical changes. Push new file contents anyway?

My minor suggestion on that is to drop the word "new" as we'll be pushing the whole file instead of just the "new bits". Other that that, it looks pretty good to me.